content/pieces/back/index.md

```diff
@@ -46,8 +46,8 @@ https://imagesnippets.com/
 ### b⃣ . e⃣ . n⃣ . n⃣ . y⃣ .
 ```
 https://www.instagram.com/twicetunatuna/
-https://newart.city
-https://gentleplants.com
+https://newart.city/
+https://gentleplants.com/
```

Comment:
lol I thought we agreed that inconsistencies == vibes

Reply:
@benhylau couldn't resist to sneak this in

Reply:
Oh I thought that didn't cross the bar for a hotfix, but if we are deploying anyways, we should fix, no?

Reply:
yeah just as always, impressed with your super attention to detail